source/codegen/service_helpers.py

```diff
@@ -16,7 +16,7 @@ def create_args(parameters):
       if is_output and is_string_arg(parameter):
         type_without_brackets = common_helpers.get_underlying_type_name(parameter['type'])
         result = f'{result}({type_without_brackets}*){parameter_name}.data(), '
-      elif parameter['type'] == 'ViBoolean[]':
+      elif parameter['type'] == 'ViBoolean[]' or parameter['type'] == 'ViSession[]':
```

Comment:
minor: Can you change it to `parameter['type'] in {"ViBoolean[]", "ViSession[]"}`

Reply:
Done.